### PR TITLE
Initial addition of separating Download from RPC Server page

### PR DESCRIPTION
**Pull request by icarusz**
As Topper suggested I changed the Download page to be separate from the RPC page and have them cross reference each other.  I decided to make "Download" a top level menu item on the left, let me know if you disagree with that.  Brings more attention.

### CODE REVIEW

File: docs/litepaper.md [hunk 1]
```diff
@@ -123,4 +123,11 @@ To reiterate, Tupelo maintains the speed of private DLT solutions which can supp
 
 ---
 
+### Interested in building on Tupelo, have comments or want to learn more?
```
Comment (tobowers) [inline docs/litepaper.md:126]: seems like this should be an include too?

File: tutorials/download/tupelo-v0.0.8-darwin-amd64-checksum .txt [hunk 2]
```diff
@@ -0,0 +1 @@
+46b5ce76a550fcfc4a0e23bdebb716ff35c2b7413b58ddaf3efb1a386018897d tupelo-v0.0.8-darwin-amd64
```
Comment (tobowers) [inline tutorials/download/tupelo-v0.0.8-darwin-amd64-checksum .txt]: there's a space in the filename here

File: download.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+***
+
+## OSX
+
+[Tupelo 0.0.8 - OSX](tutorials/download/tupelo-v0.0.8-darwin-amd64){: .btn .btn-green .fs-5 .mb-4 .mb-md-0 .mr-2 }  
+[Tupelo 0.0.8 - OSX - Checksum](tutorials/download/tupelo-v0.0.8-darwin-amd64-checksum.txt){: .btn .btn-green .fs-5 .mb-4 .mb-md-0 .mr-2 }
+
+***
+
+Looking to connect to the Test-Net rather than run locally?
```
Comment (tobowers) [inline download.md]: is this how we're going to spell test net all the time?

File: docs/litepaper.md (hunk 1 above)
Comment (icarusz) [reply docs/litepaper.md:126]: I wanted to vary the intro text in different places depending on page context.  So it was part of the include but I moved it out.

File: download.md (hunk 3 above)
Comment (icarusz) [reply download.md]: Changing to a new standard of "test network" and when conversational, optionally test net.  I will probably do a separate PR to clean this up elsewhere.

**Review by tobowers — APPROVED** [review]
👍

**Review by brandonwestcott — APPROVED** [review]

File: tutorials/download/tupelo-v0.0.8-darwin-amd64-checksum .txt (hunk 2 above)
Comment (icarusz) [reply tutorials/download/tupelo-v0.0.8-darwin-amd64-checksum .txt]: Fixorated.

**Comment by icarusz** [comment]
I think everything is addressed and don't want to see this wait too long.
I am merging unless there are any objections.

**Pull request merged**